server/executor/trigger/http.go

```diff
@@ -81,7 +82,17 @@ func (te *httpTriggerer) Trigger(ctx context.Context, test model.Test, opts *Tri
 	if tReq.Body != "" {
 		body = bytes.NewBufferString(tReq.Body)
 	}
-	req, err := http.NewRequestWithContext(ctx, strings.ToUpper(string(tReq.Method)), tReq.URL, body)
+
+	parsedUrl, err := url.Parse(tReq.URL)
+	if err != nil {
+		return response, err
+	}
+
+	if parsedUrl.Scheme == "" {
```

Comment:
We need to test an edge case: if the address comes with a `:443`, should we change this scheme to `https`?

Reply:
I think that's just a really weird use case, here's what postman shows
![Screenshot 2023-04-14 at 10 03 19](https://user-images.githubusercontent.com/11051031/232096686-6e8cef35-26a9-4ba2-91f3-661e7eb15ad0.png)

Reply:
In that case, we can live with that situation.